js/setup.js

```diff
@@ -4,11 +4,133 @@ var WIZARD_NAMES = ['Иван', 'Хуан Себастьян', 'Мария', 'К
 var WIZARD_SURNAMES = ['да Марья', 'Верон', 'Мирабелла', 'Вальц', 'Онопко', 'Топольницкая', 'Нионго', 'Ирвинг'];
 var WIZARD_COAT_COLORS = ['rgb(101, 137, 164)', 'rgb(241, 43, 107)', 'rgb(146, 100, 161)', 'rgb(56, 159, 117)', 'rgb(215, 210, 55)', 'rgb(0, 0, 0)'];
 var WIZARD_EYES_COLORS = ['black', 'red', 'blue', 'yellow', 'green'];
+var WIZARD_FIREBALL_COLORS = ['#ee4830', '#30a8ee', '#5ce6c0', '#e848d5', '#e6e848'];
 var WIZARDS_COUNT = 4;
 
+var MIN_NAME_LENGTH = 2;
+var MAX_NAME_LENGTH = 25;
+
+var ESCAPE_KEY_CODE = 27;
+var ENTER_KEY_CODE = 13;
+
 // получаю элемент DOM с классом setup и записываю его в переменную, затем удаляю у него класс hidden, чтобы он появился
-var userDialog = document.querySelector('.setup');
-userDialog.classList.remove('hidden');
+var setup = document.querySelector('.setup');
+var setupOpen = document.querySelector('.setup-open');
+var setupClose = document.querySelector('.setup-close');
```

Comment:
```suggestion
var setupClose = setup.querySelector('.setup-close');
```
https://up.htmlacademy.ru/javascript/20/criteries#d21
> Поиск элементов по селекторам делается минимальное количество раз, после этого ссылки на элементы сохраняются.